### PR TITLE
Reader: make sizing and spacing adjustments for the serif font change from Merriweather with Noto Serif

### PR DESCRIPTION
**Pull request by raoulwegat**
This PR is for changes to Reader due to Noto Serif replacing Merriweather. Branched from https://github.com/Automattic/wp-calypso/pull/12326.

### CODE REVIEW

File: client/devdocs/design/typography.jsx [hunk 1]
```diff
@@ -48,21 +48,21 @@ export default React.createClass( {
 		};
 
 		const contentTitle = {
-			fontFamily: 'Merriweather',
+			fontFamily: 'Noto Serif',
```
Comment (mtias) [inline client/devdocs/design/typography.jsx]: Why aren't these using the `$serif` var?

Comment (bluefuton) [inline client/devdocs/design/typography.jsx]: @mtias I know the `$serif` variable is available in Sass, but not sure if we can use it in JSX components?

Comment (mtias) [inline client/devdocs/design/typography.jsx]: Ah, missed it was a jsx for some reason.

**Comment by drw158** [comment]
#12326 Should already have this change included. Why this branch?

**Comment by bluefuton** [comment]
> #12326 Should already have this change included. Why this branch?

@drw158 I think @raoulwegat made this PR to do any Reader-specific spacing/sizing tweaking required as a result of the font change (following on from #12326). I'll update the title to better reflect this.

**Comment by drw158** [comment]
One note: all declarations of font-weight 300 will need to be changed to 400. I think the Reader uses Merriweather Light in some cases.

**Pull request closed without merging**